client/views/author/submissions.list.html

```diff
@@ -20,5 +20,19 @@ <h2 class="panel-title">{{s.title}}</h2>
         </div>
       </div>
     </div>
+    <div ng-show="!submissions || submissions.length == 0">
```

Comment:
Die Größe soll immer so sein wie das Formular. Aber deinen Punkt habe ich schon aufgenommen in "harmonize column width"